sample.conf

```diff
@@ -0,0 +1,12 @@
+[ANSIBLE_TOWER]
```

Comment:
@bzwei From the on premise client we will always be connecting to the cloud.redhat.com using Mutual Authentication (mTLS) so even though we are just testing locally we need the the cert and key in the sample config file. As listed here
https://github.com/mkanoor/python_catalog_mqtt_client/blob/829a510ce3681243d8669dc09305f1551e629797/config/sample.conf#L10
As we move this to the on premise we need the cert and key. The location listed in the above sample will not change in a RHEL machine. That is the default location.

Reply:
Sure. This sample conf contains parameters used by the current implementation. We will add cert and key when we actually use them for connection.